src/provider/qti.js

```diff
@@ -487,10 +487,14 @@ var qtiProvider = {
                 }
             })
             .on('pause', function(data) {
+                const context = self.getTestContext();
+
                 this.setState('closedOrSuspended', true);
 
                 this.getProxy()
                     .callTestAction('pause', {
+                        itemDefinition: context.itemIdentifier,
```

Comment:
Can you please call it `itemIdentifier`, because it is the convention and it is just an id, not the definition itself.

Reply:
this was a request from @emgolubev to call it like this

Reply:
Oh yes right! This is the name of the API parameter. Good catch.

Reply:
please be sync with https://github.com/oat-sa/extension-tao-testqti/pull/1986/files

Reply:
let's keep `itemDefinition` please